src/fides/api/task/create_request_tasks.py

```diff
@@ -422,6 +423,7 @@ def collect_tasks_fn(
         data[tn.address] = tn
 
 
+@log_context
```

Comment:
do we need to add the capture args of privacy request id here?

Reply:
In this case, this happens automatically because `PrivacyRequest` implements the `Contextualizable` interface by defining the `get_log_context` function
```
def get_log_context(self) -> Dict[LoggerContextKeys, Any]:
        return {LoggerContextKeys.privacy_request_id: self.id}
```
In other words, the `log_context` will check every function argument and call `get_log_context` automatically if it's available. `PrivacyRequest` will contribute `privacy_request_id: 123` to the logger context

Reply:
Thanks for explaining!